.github/workflows/SQL2022.yml

```diff
@@ -110,6 +110,27 @@ jobs:
       - name: Configure the R runtime installed for MLS with SQL Server
         run: C:\MLS\R\library\RevoScaleR\rxLibs\x64\RegisterRext.exe /configure /rhome:"C:\MLS\R" /instance:"MSSQLSERVER"
 
+      - name: Install Python for MLS
```

Comment:
in a yaml comment above this section, Link to the Python installation and configuration section of the SQL 22 MLS install docs. Make sure to remove the `en-us` from the URL.